client/my-sites/checkout/cart/cart-messages-mixin.jsx

```diff
@@ -31,7 +31,8 @@ module.exports = {
 	},
 
 	getChargebackErrorMessage() {
-		return this.props.translate(
+		const translate = this.props.translate || i18nCalypsoTranslate;
```

Comment:
I took a quick look and this seems to be used in 2 components, both localized:

```
$ ack -l cart-messages-mixin client server

client/my-sites/checkout/cart/popover-cart.jsx
client/my-sites/checkout/cart/secondary-cart.jsx
```

Maybe worth just refactoring those components to not use the mixin? Is this fallback even worth it?

Reply:
`SecondaryCart` also uses `observe( 'sites' )` mixin, so I won't even consider refactoring to remove mixin at this point.

Reply:
Hmm. It'd be quite nice to get rid of a mixin that's only used by 2 components.

Can you turn it into a component and just use that inside those components' `render()` methods? Akin to https://github.com/Automattic/wp-calypso/pull/18625 I mean.

Reply:
Sure, I'll give that a try